test/passenger_test.rb

```diff
@@ -68,7 +73,79 @@
     end
   end
 
+  # Test for 1.2 net_expenditures
   describe "net_expenditures" do
-    # You add tests for the net_expenditures method
+    before do
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: []
+        )
+      @driver = RideShare::Driver.new(
+        id: 3,
+        name: "Test Driver",
+        vin: "12345678912345678"
+      )
+    end
+
+    it "returns the correct total amount of money that a passenger has spent" do
+      2.times do
+        trip = RideShare::Trip.new(
+          id: 8,
+          passenger: @passenger,
+          driver: @driver,
+          start_time: "#{Time.now - 60 * 60}",
+          end_time: "#{Time.now + 60 * 60}",
+          rating: 5,
+          cost: 10
+          )
+        @passenger.add_trip(trip)
+      end
+      
+      expect (_@passenger.net_expenditures).must_equal 20
```

Comment:
Using the `_` with `expect` is unnecessary:

```suggestion
      expect(@passenger.net_expenditures).must_equal 20
```